packages/teleterm/src/ui/QuickInput/QuickInput.tsx

```diff
@@ -219,6 +220,18 @@ const Input = styled.input(props => {
   };
 });
 
+const Shortcut = styled(Box)`
+  position: absolute;
+  right: 12px;
+  top: 12px;
+  padding: 2px 3px;
+  color: rgba(255, 255, 255, 0.6);
+  background-color: rgba(255, 255, 255, 0.05);
```

Comment:
What about this?

```suggestion
  color: ${({ theme }) => theme.colors.text.secondary};
  background-color: ${({ theme }) => theme.colors.action.hover};

```

Reply:
Done, I used `theme.colors.primary.light` for background, is a bit darker than `action.hover`